GliaWidgets/View/Chat/Entry/ChatMessageEntryView.swift

```diff
@@ -21,11 +21,11 @@ class ChatMessageEntryView: UIView {
             if isChoiceCardModeEnabled {
                 textView.resignFirstResponder()
             }
-            
+
```

Comment:
Just to explain why it is good to avoid trailing whitespace - as you can see, this file has no actual changes apart from these whitespaces, but it still shows up as 24 changed lines in the pull request, making it unnecessarily big - "false changes", basically. You can read more on that in this [SO post.](https://stackoverflow.com/questions/300489/why-is-it-bad-to-commit-lines-with-trailing-whitespace-into-source-control)